apps/cli/commands/ingest_elasticsearch.py

```
@@ -0,0 +1,81 @@
+            endpoint=config.elasticsearch_url,
+            index=index,
+        )
+        normalizer = Normalizer()
+        chunker = Chunker()
+        embedder = Embedder(tei_url=config.tei_embedding_url)
+        qdrant_writer = QdrantWriter(
+            url=config.qdrant_url,
+            collection_name=config.collection_name,
+        )
+
+        console.print(f"[yellow]Loading documents from Elasticsearch...[/yellow]")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)